aemiers/rancid-tomatillos
# Feature/filter

## Pull Request

### Description

* Is this a feature or a fix?
Feature!

* Where should the reviewer start?
Please start by looking at App, Header, and Form to view grandparent-child interaction

* What functionalities do these changes affect? Why were these changes made? What was the motivation behind these changes?
Search bar should search automatically when typed into. It should also update with backspace and when the. xbutton is clicked (no refresh)

* What outside features or research did you use to implement this fix?
Stack overflow

* How should this be tested?
Please pull down the branch and make sure it actually works. I had to fix a bug where the search bar was only working onchange. Now it should work with onchange and keyup

closes #46
